test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -56,6 +56,10 @@ Describe "Import-Module" -Tags "CI" {
         Import-Module TestModule -RequiredVersion 1.1
         (Get-Module TestModule).Version | Should -BeIn "1.1"
     }
+
+    It "should throw if 'MaximumVersion' is less than 'Version' when using -FullyQualifiedName parameter" {
```

Comment:
In addition to this test, we should add a more specific one using the `[ModuleSpecification]` constructor.

You should find those tests [here](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Module/ModuleSpecification.Tests.ps1) (I know that's not an obvious place at all).

Note to self: once https://github.com/PowerShell/PowerShell/pull/7499 is merged, this test will be included in that.

Reply:
Let me know if you need any help on creating that test btw, I'm always happy to

Reply:
Done.